lib.sh

```diff
@@ -1425,6 +1426,22 @@ do
 done
 }
 
+# Use this to send system mails
+# e.g.: send_mail "subject" "text"
+send_mail() {
+    local RECIPIENT
+    if [ -f /etc/msmtprc ]
+    then
+        RECIPIENT=$(grep "recipient=" /etc/msmtprc)
+        RECIPIENT="${RECIPIENT##*recipient=}"
+        if [ -n "$RECIPIENT" ]
+        then
+            print_text_in_color "$ICyan" "Sending '$1' to $RECIPIENT"
+            echo -e "$2" | mail --subject "NcVM - $1" "$RECIPIENT"
+        fi
+    fi
+}
+
```

Comment:
BTW: I am voting to leave this in here, because you can then properly test if this works.

It will only send a mail if smtp mail was setup.

Reply:
Sure.